docs/configure.rst

```diff
@@ -0,0 +1,111 @@
+.. highlight:: console
+
+
+.. _configure:
+
+=========
+Configure
+=========
+
+This page shows you how to install `Croud`_, the dedicated command-line
+interface (CLI) and log in to CrateDB Cloud. Croud lets you interact with
```

Comment:
"page" -> "document"

drop "dedicated"

italicize "command-line interface"

"interface (CLI) for interacting with CrateDB Cloud"

then say something like "once Croud is installed, you can log in and ..."